src/app/pages/about/about.component.scss

```
@@ -4,303 +4,146 @@ $padding: 11%;
 
 @media only screen and (max-width: 500px) {
 
-  .about-container .content{
-    display: flex;
-    flex-direction: column;
-
+  .content{
     .logo-alfy {
       max-height: 240px !important;
       width: auto;
-      margin: 0 auto !important;
       top: 0 !important;
     }
   }
+
+  .low-mock-up {
+    display: flex !important;
+    flex-direction: column !important;
+  }
+
 }
 
+  h2 {
+    font-size: 30px;
+  }
+
 .about-container {
   display: flex;
   flex-direction: column;
+  background-color: $accent-lighter;
 
-  .row {
-    display: flex;
-    padding: 0 $padding;
-    align-content: center;
-    flex-direction: column;
-
-    .title {
-      margin: 25px 0;
-
-      h2 {
-        margin: 0;
-      }
-    }
-
-    .content {
-      margin: 0 0 25px;
-      line-height: 23px;
-      font-size: 17px;
-    }
+  .caroussel {
+    //margin: 2%;
+    border-radius: 3%;
+  }
 
     &.alfy-presentation {
       background-color: white;
-
-      .main-title {
-        position: relative;
-        display: flex;
-        justify-content: flex-end;
-        flex-flow: column;
-        height: 500px;
-        width: 100% * ( 1 / 0.78 );
-        left: 11% * ( -1 / 0.78 );
-
-        &.no-background {
-          background-image: url('/assets/img/fustel.jpg');
-          background-size: cover;
-          background-color: $accent-lighter;
-          background-repeat: no-repeat;
-          background-position: center center;
-        }
-
-        .voile {
-          position: relative;
-          height: 100%;
-          display: flex;
-          align-items: flex-end;
-          padding-bottom: 20px;
-          padding-right: 50%;
-          width: 100%;
-          font-weight: bolder;
-          box-sizing: border-box;
-          overflow: hidden;
-
-          &:before {
-            content: "";
-            display: block;
-            position: absolute;
-            left: 0;
-            bottom: 0;
-            height: 0;
-            width: 0;
-            border-bottom: 330px solid rgba(255,255,255,0.7);
-            border-right: 800px solid transparent;
-            z-index: 0;
-          }
-
-          h1 {
-            width: 100%;
-            z-index: 5;
-            min-width: 260px;
-            max-width: 370px;
-            font-size: 46px;
-            line-height: 52px;
-            font-family: Bitter, serif;
-            margin: 0 20% 0 7%;
-          }
-        }
-      }
-
-      .logo-alfy {
-        position: relative;
-        top: -25px;
-        max-height: 230px;
-        margin: -15px 8% 0 25px;
-      }
     }
 
-    &.goals-container {
+    .low-mock-up {
       display: flex;
-      flex-direction: column;
+      flex-direction: row;
 
-      h3 {
-        margin-bottom: 30px;
-      }
+      .agenda {

```

Pour fixer la hauteur de l'agenda en fonction de la hauteur du reste de la page (présentation, partenaire....), partie 1

```suggestion
      position: relative;
      .agenda {
        @media only screen and (min-width: 768px) {
          position: absolute;
          top: 0;
          bottom: 0;
          left: 0;
          right: 64%;
        }
```